lib/auth/api.go

```diff
@@ -704,6 +704,8 @@ type ReadDiscoveryAccessPoint interface {
 
 	// GetDatabases returns all database resources.
 	GetDatabases(ctx context.Context) ([]types.Database, error)
+	// GetDatabase returns a database resource with the given name if it exists.
+	GetDatabase(ctx context.Context, name string) (types.Database, error)
```

Comment:
Is this a customer-facing API? If so, does the next release need to bump the minor version instead of just patch?

Reply:
No. This is just an interface extension. the underlying stuff already implements that API